fdbserver/BlobWorker.actor.cpp

```diff
@@ -2599,11 +2607,32 @@ ACTOR Future<Void> handleRangeRevoke(Reference<BlobWorkerData> bwData, RevokeBlo
 	}
 }
 
+void handlePruneRequest(Reference<BlobWorkerData> bwData, PruneFilesRequest req) {
+	// if we see an out of order prune req, just skip it because we would have
+	// pruned everything before this version when we handled latestPruneVersion
+	if (req.pruneVersion <= bwData->latestPruneVersion) {
+		return;
```

Comment:
Hmm, yeah I don't think this is possible since GRV's never drift backwards _and_ since a request has to be ACK'd before a new prune request would be sent (there won't ever be multiple prune requests in flight to the same worker).

Reply:
Makes sense.
We can always assert on it though to be sure